Gemfile

```diff
@@ -8,8 +8,12 @@ ruby '3.0.2'
 gem 'rails', '~> 6.1.5'
 
 gem 'bootsnap', '>= 1.4.4', require: false # https://rubygems.org/gems/bootsnap
+gem 'bcrypt'
+gem 'discard', '~> 1.2'
 gem 'dotenv-rails', '~> 2.7', '>= 2.7.6' # https://rubygems.org/gems/dotenv-rails
+gem "image_processing", ">= 1.2"
```

Comment:
Why did you decide to add image_processing?

Reply:
I have understood that it is necessary for image analysis and processing.

Reply:
Ok, I understand but in this case, we have chosen to use Active Storage for file processing.